STANDARD.md

```diff
@@ -448,22 +448,22 @@ short-hand notation (see: *Constant Record Components*).
                               with `constant record components`
     - example: reading example (with h5py) in Python:
 ```python
-def is_const_record(record_name, component_name) :
-    return ("value" in record_name["component_name"].attrs.keys())
+def is_const_record(record, component_name) :
+    return ("value" in record[component_name].attrs.keys())
 
-def get_component(record_name, component_name) :
-    if is_const_record(record_name, component_name) :
-        return record_name["component_name"].attrs["value"]
+def get_component(record, component_name) :
+    if is_const_record(record, component_name) :
+        return record[component_name].attrs["value"]
     else :
-        record_name["component_name"][:]
+        return record[component_name].value
```

Comment:
ah, interesting. I never used `.value` on `h5py.Dateset` types before. It returns the `numpy.ndarray` directly, cool.
yes, that is way more general then the example assuming a 1D array `[:]`.